### PR TITLE
ADDED NEW H1 TAG

### PR DESCRIPTION
**Pull request by necr0coder**
i added a h1 tag

### CODE REVIEW

**Comment by jrhurleycode** [comment]
ok I'll check it out later.

**Review by jrhurleycode — COMMENTED** [review]
ehh not feeling this

File: index.html [hunk 1]
```diff
@@ -8,5 +8,6 @@
 <body>
     <h1>Hello World!</h1>
     <h2>Hello again!</h2>
+    <h1>ONE MORE TIME</h1>
```
Comment (jrhurleycode) [inline index.html]: change this header.  sounds dumb

File: index.html [hunk 2]
```diff
@@ -8,6 +8,6 @@
 <body>
     <h1>Hello World!</h1>
     <h2>Hello again!</h2>
-    <h1>ONE MORE TIME</h1>
+    <h1>....hello again</h1>
```
Comment (jrhurleycode) [inline index.html]: get rid of the periods dummy

**Review by jrhurleycode — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (necr0coder) [reply index.html]: ok i did

Comment (jrhurleycode) [reply index.html]: thank you slave <3

**Review by jrhurleycode — APPROVED** [review]

**Pull request merged**